contentcuration/contentcuration/templates/updates.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE HTML>
```

Comment:
Heh, I bet we have this issue on all the templates but looks like this is duplicated in `base.html`